src/components/Title.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Title = () => {
+  return (
+    <div class="font-russo" style={{ fontSize: "110px", color: "#FFFFFF" }}>
```

Comment:
you want to use tailwind classes, instead of regular css
checkout the documentation:
https://tailwindcss.com/docs/text-color